<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="BCT-LTP-G1007" failures="1" tests="1" name="testng" time="7.949" errors="0" timestamp="2023-03-14T15:45:14 IST">
  <testcase classname="com.blazeclan.qa.test.testng.BaseTest" name="@BeforeTest setUp" time="7.298">
    <failure type="org.openqa.selenium.remote.http.ConnectionFailedException" message="Unable to establish websocket connection to http://localhost:50208/devtools/browser/e0a289a0-fb0a-49b4-a830-6a948ff1a4ca
Build info: version: &amp;apos;4.7.0&amp;apos;, revision: &amp;apos;0a5b49d16f&amp;apos;
System info: os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;17.0.6&amp;apos;
Driver info: driver.version: ChromeDriver">
      <![CDATA[org.openqa.selenium.remote.http.ConnectionFailedException: Unable to establish websocket connection to http://localhost:50208/devtools/browser/e0a289a0-fb0a-49b4-a830-6a948ff1a4ca
Build info: version: '4.7.0', revision: '0a5b49d16f'
System info: os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '17.0.6'
Driver info: driver.version: ChromeDriver
at org.openqa.selenium.remote.http.netty.NettyWebSocket.<init>(NettyWebSocket.java:102)
at org.openqa.selenium.remote.http.netty.NettyWebSocket.lambda$create$3(NettyWebSocket.java:128)
at org.openqa.selenium.remote.http.netty.NettyClient.openSocket(NettyClient.java:107)
at org.openqa.selenium.devtools.Connection.<init>(Connection.java:77)
at org.openqa.selenium.chromium.ChromiumDriver.lambda$new$2(ChromiumDriver.java:116)
at java.base/java.util.Optional.map(Optional.java:260)
at org.openqa.selenium.chromium.ChromiumDriver.<init>(ChromiumDriver.java:114)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:81)
at org.openqa.selenium.chrome.ChromeDriver.<init>(ChromeDriver.java:49)
at com.blazeclan.qa.utility.CrossBrowser.selectDriver(CrossBrowser.java:28)
at com.blazeclan.qa.base.CommonFunctions.invokeBrowser(CommonFunctions.java:52)
at com.blazeclan.qa.test.testng.BaseTest.setUp(BaseTest.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:643)
at org.testng.TestRunner.beforeRun(TestRunner.java:632)
at org.testng.TestRunner.run(TestRunner.java:594)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @BeforeTest setUp -->
  <testcase classname="com.blazeclan.qa.test.testng.BaseTest" name="@AfterTest logout" time="0.0">
    <skipped/>
  </testcase> <!-- @AfterTest logout -->
  <testcase classname="com.blazeclan.qa.test.testng.LoginTest" name="LoginPage" time="0.555">
    <failure type="java.lang.ExceptionInInitializerError">
      <![CDATA[java.lang.ExceptionInInitializerError
at com.blazeclan.qa.logging.Log.extentLog(Log.java:31)
at com.blazeclan.qa.logging.Log.info(Log.java:16)
at com.blazeclan.qa.utility.TestListener.onTestStart(TestListener.java:31)
at org.testng.internal.TestListenerHelper.runTestListeners(TestListenerHelper.java:106)
at org.testng.internal.invokers.TestInvoker.runTestResultListener(TestInvoker.java:277)
at org.testng.internal.invokers.TestInvoker.registerSkippedTestResult(TestInvoker.java:829)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:604)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.remote.RemoteWebDriver.getCapabilities()" because "com.blazeclan.qa.reports.ExtentManager.driver" is null
at com.blazeclan.qa.reports.ExtentManager.createExtentReports(ExtentManager.java:28)
at com.blazeclan.qa.reports.ExtentTestManager.<clinit>(ExtentTestManager.java:12)
... 29 more
]]>
    </failure>
  </testcase> <!-- LoginPage -->
  <testcase classname="com.blazeclan.qa.test.testng.BaseTest" name="@BeforeTest setUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setUp -->
  <testcase classname="com.blazeclan.qa.test.testng.BaseTest" name="@BeforeTest setUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setUp -->
  <testcase classname="com.blazeclan.qa.test.testng.BaseTest" name="@AfterTest logout" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest logout -->
  <testcase classname="com.blazeclan.qa.test.testng.BaseTest" name="@BeforeTest setUp" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest setUp -->
  <testcase classname="com.blazeclan.qa.test.testng.BaseTest" name="@AfterTest logout" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest logout -->
  <testcase classname="com.blazeclan.qa.test.testng.BaseTest" name="@AfterTest logout" time="0.0">
    <ignored/>
  </testcase> <!-- @AfterTest logout -->
</testsuite> <!-- testng -->
